examples/topology/generateVoxelComplexTables.cpp

```diff
@@ -0,0 +1,105 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+/**
+ * @file generateVoxelComplexTables.cpp
+ * @ingroup Examples
+ * @author Pablo Hernandez-Cerdan (\c pablo.hernandez.cerdan@outlook.com)
+ * Institute of Fundamental Sciences. Massey University.
+ * Palmerston North, New Zealand
+ *
+ * @date 2016/03/20
+ *
+ * Creates precomputed tables for all possible configurations of the
+ * neighborhood of a voxel in 3D with a topology of 26 neighbors.
+ * The table is a map between all this configurations and the result of applying
+ * an input selected predicate function to a center voxel.
+ * The options for the functions are the Skel function from
```

Comment:
a bit too generic right now..

Reply:
Updated the description to be more specific to isthmus